lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -140,6 +144,97 @@ class AddNewPrimUndoableCommand : public Ufe::UndoableCommand
     PXR_NS::TfToken _primToken;
 };
 
+const char* selectUSDFileScript = R"(
+global proc string SelectUSDFileForAddReference()
+{
+    string $result[] = `fileDialog2 
+        -fileMode 1
+        -caption "Add Reference to USD Prim"
+        -fileFilter "USD Files (*.usd *.usda *.usdc);;*.usd;;*.usda;;*.usdc"`;
```

Comment:
It's my understanding that .usdz should not be in the list of files that can be referenced, so just adding the other three here..